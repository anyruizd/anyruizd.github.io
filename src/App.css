.App {
  background-color: rgba(248, 211, 42, 0.3);
  display: flex;
  height: 100vh;
  width: 100vw;
}

.content {
  align-items: center;
  background-color: rgb(226, 149, 25);
  border-radius: 5%;
  cursor: pointer;
  display: flex;
  height: 250px;
  justify-content: center;
  margin: 30vh auto;
  transition: transform 0.5s;
  width: 250px;
}

.content:hover {
  transform: scale(1.1);
}

.content p {
  color: white;
  font-family: Helvetica, sans-serif;
  font-size: 24px;
  text-align: center;
}