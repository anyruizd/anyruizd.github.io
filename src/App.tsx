import React, { Fragment } from 'react';
import styled, { createGlobalStyle } from 'styled-components';
import { COLORS } from './constants';
import { Header } from './components/Header';

const Normalize = createGlobalStyle`
  * {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
  }
`;

const AppWrapper = styled.div`
  background-color: ${COLORS.darkBackground};
  display: flex;
  height: 100vh;
  width: 100vw;
`;

function App() {
  return (
    <Fragment>
      <Normalize />
      <AppWrapper>
        <Header />
      </AppWrapper>
    </Fragment>
  );
}

export default App;
