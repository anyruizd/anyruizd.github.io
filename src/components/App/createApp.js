/**
 * Creates the application
 * @param {Object} node - HTML element
 */
export function createApp (node) {
  node.innerHTML = '<p> ✨ Any\'s site is under construction ✨</p>';
}