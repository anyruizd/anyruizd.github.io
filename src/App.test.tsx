import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

test("renders any's site warning", () => {
  const { getByText } = render(<App />);
  const textElement = getByText(/Any's site is under construction/i);
  expect(textElement).toBeInTheDocument();
});
