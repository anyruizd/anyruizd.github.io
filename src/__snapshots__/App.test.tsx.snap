// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <div
    class="sc-AxiKw eGgNkt"
  >
    <header
      class="sc-AxirZ iwGIPQ"
    >
      <div
        class="sc-AxjAm gwyFZa toggle-container menu-closed"
      >
        <span
          class="line line-top"
        />
        <span
          class="line line-mid"
        />
        <span
          class="line line-bottom"
        />
      </div>
    </header>
  </div>
</div>
`;
